project/REVIEWING.md

```diff
@@ -0,0 +1,170 @@
+Pull request reviewing process
+==============================
+
+# Workflow
+
+A opened pull request can be in 1 of 5 distinct states, for each of which there is a corresponding
+label that needs to be applied:
+
+## Triage - `status/0-triage`
+
```

Comment:
Might be useful to say that this label is automatically applied as new PRs are opened - just so people know they don't need to manually do it themselves.